modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -17,7 +17,7 @@ Install the rtfctl utility to manage Runtime Fabrics locally, with support for t
 [WARNING]
 You may need root privileges to a controller node running Runtime Fabric to follow this procedure.
 
-. Download the https://runtime-fabric.s3.amazonaws.com/rtfctl/rtfctl-0.2.32[latest version] of the `rtfctl` utility.
+. Download the latest version of the `rtfctl` utility following the instructions provided in xref:install-create-rtf-arm#create-a-runtime-fabric-using-runtime-manager[Create a Runtime Fabric using Runtime Manager].
```

Comment:
The link for this is actually a separate "Downloads" page in Runtime Manager, under the Runtime Fabrics tab.
Something like ".. by navigating to the Downloads page under the Runtime Fabrics tab in Anypoint Runtime Manager."

Reply:
Got it - thank you!